tests/test_data/prov/metadata/director/root.json

```diff
@@ -1,64 +1,77 @@
 
 {
-	"_type" : "Root",
```

Comment:
Hold on, aren't these here to check that the migrations are backwards compatible?

Reply:
No, we just move everything we have from file system to sql and in comments above I wrote that I will add the old format support. So everything should work.

Reply:
but isn't the purpose of the metadata in `tests/test_data/prov/metadata` to be a sample 'old metadata' snapshot to test that backwards compatibility is maintained?

My understanding is that  `tests/test_data/prov/metadata` is there to mimic a old device just coming back online, and to check that everything still works.  Is that wrong?

Reply:
Yes You are right, I just reverted to old metadat, and evrything works well